extension/src/test/runTest.ts

```diff
@@ -6,7 +6,8 @@ async function main() {
 	try {
 		// The folder containing the Extension Manifest package.json
 		// Passed to `--extensionDevelopmentPath`
-		const extensionDevelopmentPath = path.resolve(__dirname, '../../../../');
+		// The __dirname is  \AutoLispExt\out\test
+		const extensionDevelopmentPath = path.resolve(__dirname, '../../');
```

Comment:
This was handled in PR #160 and quite a bit more. Make sure you update to the latest before these test additions get added.